### DISCUSSION TITLE
restart the wizard

### DISCUSSION THREAD

**Discussion by ckuntze** [jomjol/AI-on-the-edge-device#1267] [General]
Can you restart the wizard that is run on the first start?

The ESP is in a hard-to-reach position

**Comment by caco3** [comment]
1. You must go to Settings > Configuration 
1. Then check the box "Expert Mode"
2. Then click on the showing up button "Edit config.ini directly"
3. Then scoll down and set `SetupMode` to `true`
4. Save and restart
5. When the main page gets loaded, press F5 to refresh and show the Wizard page.

**Comment by ckuntze** [comment]
you make my day....

thanks
